src/qtcore/qml/QMLEngine.js

```diff
@@ -115,7 +115,6 @@ QMLEngine = function (element, options) {
 
         // Create and initialize objects
         var component = new QMLComponent({ object: tree, parent: null });
-
```

Comment:
Unrelated style change, let's keep those minimal until we finish with the merges..